test/hmacKey.ts

```diff
@@ -198,21 +203,9 @@ describe('HmacKey', () => {
       });
     });
 
-    describe('update', () => {
-      it('should throw without metadata object', () => {
-        assert.throws(() => {
-          hmacKey.update(undefined, assert.ifError);
-        }, /Cannot update HmacKey/);
-      });
-
-      it('should throw with an empty metadata object', () => {
-        assert.throws(() => {
-          hmacKey.update({}, assert.ifError);
-        }, /Cannot update HmacKey/);
-      });
-
-      it('should accept an options object', () => {
-        hmacKey.update({state: 'INACTIVE'}, {}, assert.ifError);
+    describe('setMetadata', () => {
```

Comment:
No tests needed! The common tests cover that for us, all we need to know is that we properly defined the configuration object in the constructor.

Reply:
Same for all of these methods, actually.

Reply:
Removed, thank you!